tests/campaigns/test_integration.py

```diff
@@ -249,3 +250,19 @@ def setUp(self):
 
     def test_downgrades_subscription(self):
         self.assert_sub_returned(self.result, self.campaign_id, 100)
+
+
+class TestIterSubscriptions(BaseCampaignTestCase):
+    def setUp(self):
+        self.subscription = Subscription.iter_subscriptions().next()
+
+    def test_returns_iterator_over_subscription_objects(self):
+        self.assertIsInstance(self.subscription, Subscription)
+
+
+class TestListSubscriptions(BaseCampaignTestCase):
+    def setUp(self):
+        self.subscriptions = self.service.list_subscriptions(0, 1)
+
+    def test_limits_returned_result_to_given_limit(self):
+        self.assertEqual(len(self.subscriptions), 1)
```

Comment:
What do you think about combining these two test cases into one, with a `setUpClass` to reduce test run time?